jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerCredentialHelper.java

```diff
@@ -177,7 +178,8 @@ private Credential retrieve(List<String> credentialHelperCommand)
 
       // Checks if the failure is due to a nonexistent credential helper CLI.
       if (ex.getMessage().contains("No such file or directory")
-          || ex.getMessage().contains("cannot find the file")) {
+          || ex.getMessage().contains("cannot find the file")
```

Comment:
Ccan we not detect this earlier? Or are we relying on path information that we may not have?

Reply:
Yeah, we want to test if the OS can find the command through `$PATH`. And most of the time, it isn't a full path in practice.

Reply:
Oh, that reminded me that we check the file existence if it is a path. There's another place I need to fix.

Reply:
Done with the latest commit a409b84.